website/source/docs/platform/k8s/helm/examples/standalone-load-balanced-ui.html.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: "docs"
+page_title: "Standalone Server with Load Balanced UI"
+sidebar_current: "docs-platform-k8s-examples-standalone-load-balanced-ui"
+sidebar_title: "Standalone Server with Load Balanced UI"
+description: |-
+  Describes how to set up a standalone Vault with a load balanced UI
+---
+
+# Standalone Server with Load Balanced UI
+
+~> **Important Note:** This chart is not compatible with Helm 3. Please use Helm 2 with this chart.
+
+The below `values.yaml` can be used to set up a single server Vault cluster with a LoadBalancer to allow external access to the UI and API.
+
+```yaml
+global:
+  enabled: true
+  image: "vault:1.3.0"
```

Comment:
Old version

Reply:
Also, this doesn't exist anymore.  It's now under `server`

Reply:
Removed